src/pages/[platform]/build-a-backend/data/enable-logging/index.mdx

```diff
@@ -0,0 +1,98 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Enable logging for your data resource',
+  description: 'Enable logging for your data resource',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+You can enable logging to debug your GraphQL API using CloudWatch logs.
+
+## Default Logging
+
+Default logging can be enabled by setting the `logging` property to `true` in the call to `defineData`. For example:
+
+```ts title="amplify/data/resource.ts"
+export const data = defineData({
+  some_schema,
+  some_authorizationModes,
+  logging: true
+});
+```
+
+Using `logging: true` applies the default configuration:
+- `excludeVerboseContent: true`
+- `fieldLogLevel: 'none'`
+- `retention: '1 week'`
+
+## Custom Logging
+
+You can override individual fields by providing a `DataLogConfig` object. For example:
+
+```ts title="amplify/data/resource.ts"
+export const data = defineData({
+  some_schema,
+  some_authorizationModes,
+  logging: {
+    excludeVerboseContent: false,
+    fieldLogLevel: 'all',
+    retention: '1 month'
+  }
+});
+```
+
+<Callout warning>
+**WARNING**: Setting `excludeVerboseContent` to `false` logs full queries and user parameters, which can contain sensitive data. Ensure your IAM policies limit access appropriately.
+</Callout>
+
+## Configuration Properties
+
+### `logging`
+- `true`: Enables default logging.
+- `DataLogConfig` object: Overrides one or more default fields.
+
+### `DataLogConfig` Fields
+
+1. **`retention?: LogRetention`**  
```

Comment:
`LogRetention` isn't defined in this doc. Link to its API reference definition (presumably, AppSync docs)